models/pancreas_ct_dints_segmentation/docs/README.md

```diff
@@ -74,6 +74,9 @@ Users can install Graphviz for visualization of searched architectures (needed i
 
 ![Example of Searched Architecture](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_net_arch_search_segmentation_searched_arch_example_1.png)
 
+### System memory requirement
+[The provided search architecture and checkpoints](https://github.com/dongyang0122/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/large_files.yml) result from training conducted using 400GB of system RAM. Ample RAM is essential as we aim to cache all pre-processed data points in RAM to enhance the efficiency of model searching and training with [CacheDataset](https://docs.monai.io/en/stable/data.html#cachedataset). Should any errors occur during data pre-processing or caching in model search or training, kindly decrease the caching rate `cache_rate` in the configurations (e.g., [training](https://github.com/dongyang0122/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/configs/train.yaml) and [multi-GPU training](https://github.com/dongyang0122/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/configs/multi_gpu_train.yaml)) to minimize the GPU RAM requirements.
```

Comment:
Thanks @dongyang0122 for the updates.
As for the line:
https://github.com/Project-MONAI/model-zoo/pull/406/files#diff-21a8c44891c2467d971a975c73d4488b87963f0366b003847fea6d959c4521adL42
Do you think it should be removed or modified?

Reply:
I removed that line to avoid confusion.

Reply:
Thanks, may need @zephyrie 's double confirm of the readme format.

Reply:
How can we get the confirmation?